backend/simple/queryer.go

```diff
@@ -22,9 +22,9 @@ func (q *queryer) Exec(ctx context.Context, qry string, vs ...interface{}) (sql.
 }
 
 func (q *queryer) QueryRow(ctx context.Context, qry string, vs ...interface{}) sql.Scanner {
-	return q.QueryRowContext(ctx, qry, vs...)
+	return q.QueryRowContext(ctx, qry, sql.StripOptions(vs)...)
```

Comment:
Maybe adding a integration test for this would be a good idea, i.e. testing the options passed for all the operation are trimmed when passed to the std driver ( tested by making sure the query returns what is expected).